frontend/web/components/feature-page/FeatureFilters.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react'
+import ClearFilters from 'components/ClearFilters'
+import TableSearchFilter from 'components/tables/TableSearchFilter'
+import TableTagFilter from 'components/tables/TableTagFilter'
+import TableValueFilter from 'components/tables/TableValueFilter'
+import TableOwnerFilter from 'components/tables/TableOwnerFilter'
+import TableGroupsFilter from 'components/tables/TableGroupsFilter'
+import TableFilterOptions from 'components/tables/TableFilterOptions'
+import TableSortFilter, { SortValue } from 'components/tables/TableSortFilter'
+import { getViewMode, setViewMode, ViewMode } from 'common/useViewMode'
+import { isEqual } from 'lodash'
+import Utils from 'common/utils/utils'
+import { TagStrategy } from 'common/types/responses'
+import Format from 'common/utils/format'
+
+export type FiltersValue = {
+  search: string | null
+  releasePipelines: number[]
+  page: number
+  tag_strategy: TagStrategy
+  tags: (number | string)[]
+  is_archived: boolean
+  value_search: string | null
+  is_enabled: boolean | null
+  owners: number[]
+  group_owners: number[]
+  sort: SortValue
+}
+
+type Props = {
+  value: FiltersValue
+  onChange: (next: FiltersValue) => void
+  projectId: string | number
+  orgId?: number
+  isLoading?: boolean
+}
+
+const DEFAULTS: FiltersValue = {
+  group_owners: [],
+  is_archived: false,
+  is_enabled: null,
+  owners: [],
+  page: 1,
+  releasePipelines: [],
+  search: '',
+  sort: { label: 'Name', sortBy: 'name', sortOrder: 'asc' },
+  tag_strategy: 'INTERSECTION',
+  tags: [],
+  value_search: '',
+}
+
+const sortToHeader = (s: any) => {
+  if (!s) return DEFAULTS.sort
+  if ('sortBy' in s) return s
+  return {
+    label: s.label || DEFAULTS.sort.label,
+    sortBy: s.value || DEFAULTS.sort.sortBy,
+    sortOrder: DEFAULTS.sort.sortOrder,
+  } as FiltersValue['sort']
+}
+
+// Converts filters to url params, excluding ones that are already default
+export function getURLParamsFromFilters(f: FiltersValue) {
+  const existing = Utils.fromParam() as Record<string, string | undefined>
+
+  return {
+    ...existing,
+    group_owners: f.group_owners?.length ? f.group_owners.join(',') : undefined,
+    is_archived: f.is_archived ? 'true' : undefined,
+    is_enabled:
+      f.is_enabled === null ? undefined : f.is_enabled ? 'true' : 'false',
+    owners: f.owners?.length ? f.owners.join(',') : undefined,
+    page: f.page !== DEFAULTS.page ? String(f.page) : undefined,
```

Comment:
Mmh, so it's not from this PR but testing the pagination. It seems that when we use the `< >` to navigate, we don't update the text: (100-150 out of XXX) and the current page. I don't know if you think it's straightforward to fix otherwise we can create a ticket